### DISCUSSION TITLE
Support for Safe Updates

### DISCUSSION THREAD

**Discussion by cgual206** [cloudbees/zendesk-java-client#627] [Ideas]
I was wondering if there were any plans for the Zendesk Client to support safe updates as described in the [Protecting against ticket update collisions](https://developer.zendesk.com/documentation/ticketing/managing-tickets/creating-and-updating-tickets/#protecting-against-ticket-update-collisions) documentation?

**Comment by PierreBtz** [comment]
@btpnlsl that looks like a good addition, PR welcome!

**Reply by cgual206** [reply]
@PierreBtz PR opened: https://github.com/cloudbees-oss/zendesk-java-client/pull/630

**Comment by PierreBtz** [comment]
https://github.com/cloudbees-oss/zendesk-java-client/issues/629
